### PR TITLE
[INLONG-7191][DataProxy] Remove unused code

### LINKED ISSUE #7191: [Improve][DataProxy] Remove unused code
### Description

_No response_

### InLong Component

InLong DataProxy

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by woofyzhao**
- Fixes #7191

### Motivation ###

The new unified MQ sink package path is now _org.apache.inlong.dataproxy.sink.mq_

### CODE REVIEW

File: inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/EventStat.java [hunk 1]
```diff
@@ -1,81 +0,0 @@
...
- * limitations under the License.
- */
-
-package org.apache.inlong.dataproxy.sink;
-
-import java.util.concurrent.atomic.AtomicInteger;
-import org.apache.flume.Event;
-import org.apache.inlong.dataproxy.utils.MessageUtils;
-
-public class EventStat {
```
Comment (healchow) [inline inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/EventStat.java:24]: Do other alternatives already support these deleted codes?

Comment (woofyzhao) [reply inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/EventStat.java:24]: Yes, the new unified mq sink layer has moved to sink/mq package dir

**Review by healchow — APPROVED** [review]
LGTM

**Review by luchunliang — APPROVED** [review]
LGTM

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**